### DISCUSSION TITLE
How to get the value of a custom child of a form.item?

### DISCUSSION THREAD

**Discussion by kopticx** [ant-design/ant-design#41943] [General]
### What problem does this feature solve?

I am trying to obtain the value of a component called ViewMap which is nothing more than a custom component that selects a mapbox coordinate, when I give it initial status with a Marker it takes it well and does not return that it is required, otherwise Even though I select a coordinate, I can't get the value, it tells me that it's required. I think it would be a good idea to have the component more documented for these kinds of ideas that may appear. 

### What does the proposed API look like?

More examples with custom components

**Comment by bombillazo** [comment]
Here is an example of the API:
https://ant.design/components/form#components-form-demo-customized-form-controls

And under the `Form.Item` props table you have some additional descriptions on how to do a custom input:
https://ant.design/components/form#formitem
